lib/winston-graylog2.js

```diff
@@ -53,6 +53,9 @@ function prepareMeta(meta) {
     });
   }
 
+  // merge the meta data with the statically configured metadata
+  meta = _.merge(meta, this.staticMeta);
```

Comment:
Id rather add a parameter tot he function it's self: not been a method of the object is uncleart why we;d have a `this` invocation here